maintainers.txt

```diff
@@ -0,0 +1,3 @@
+gnott
+lsh-0
+nlisgo
```

Comment:
I'd drop nathan of this list, as the repo is python so best leave it to graham and luke